var/spack/repos/builtin/packages/graphviz/package.py

```diff
@@ -29,11 +29,15 @@
 
 class Graphviz(AutotoolsPackage):
     """Graph Visualization Software"""
-    homepage = 'http://www.graphviz.org'
+
+    homepage = "http://www.graphviz.org"
+    url      = "http://www.graphviz.org/pub/graphviz/stable/SOURCES/graphviz.tar.gz"
```

Comment:
I fixed the `flake8` errors. Maybe I am missing the obvious, but how are we going to select among the two versions below, given that `url` seems not to have any version information?